pubspec.yaml

```diff
@@ -10,7 +10,7 @@ environment:
 dependencies:
   visibility_detector: ">=0.2.0 <1.0.0"
   flutter_hooks: ">=0.16.0 <1.0.0"
-  http: ">=0.13.1 <1.0.0"
+  http: ">=0.13.1 <2.0.0"
```

Comment:
by doing that you force Dart 3 as minimum dart version, so environment need to be updated too. 
And `http: ">=1.0.0 <2.0.0"`

Reply:
Do I understand this correctly, that you would prefer `http: ">=1.0.0 <2.0.0"` so the user is forced to update to Dart 3?

Because `http: ">=0.13.1 <2.0.0"` will work with both Dart 2 and Dart 3.

Reply:
Never mind I read to fast it's perfect like this sorry ˆˆ